### DISCUSSION TITLE
Cant import accounts from manifest.json windows

### DISCUSSION THREAD

**Discussion by folvos** [dyc3/steamguard-cli#209] [Q&A] [answered]
ERROR - An unexpected error occurred during manifest migration: Failed to load some accounts: [Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2))]

Caused by:
    Failed to load some accounts: [Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2)), Unknown(The system cannot find the file specified. (os error 2))]

**Comment by dyc3** [comment]
Can you post the output when you have debug logging on? Add the argument `-v debug` to your command.

**Comment by folvos** [comment]
INFO - reading manifest from C:\Users\Administrator\AppData\Roaming\steamguard-cli/maFiles
DEBUG - loading manifest: "C:\\Users\\Administrator\\AppData\\Roaming\\steamguard-cli/maFiles\\manifest.json"
INFO - Migrating manifest
DEBUG - deserializing manifest: version null

**Reply by dyc3** [reply, marked as answer]
Is that everything? I was expecting to see more logs.

`The system cannot find the file specified.` means that steamguard-cli failed to read a file because the file requested does not exist. steamguard-cli expects `maFile`s to be found in the same directory as `manifest.json`. According to these logs, your maFiles should be in `C:\Users\Administrator\AppData\Roaming\steamguard-cli/maFiles`, right? Make sure this folder has a `manifest.json` file.
